src/Service/Startup.cs

```diff
@@ -174,7 +175,16 @@ public void ConfigureServices(IServiceCollection services)
             //Enable accessing HttpContext in RestService to get ClaimsPrincipal.
             services.AddHttpContextAccessor();
 
-            ConfigureAuthentication(services, configProvider);
+            // Use HotReload aware authentication configuration when in development mode
```

Comment:
What is `HotReload aware authentication configuration`?

Reply:
I used that phrasing to describe the purpose of the `ConfigureAuthenticationV2()` function. That function wires up all authentication methods unconditionally. Today, we conditionally wire up authentication methods. 

Authentication method (scheme) configuration must occur at startup. This PR adds the newly referenced "V2" function which wires up all the schemes together. Then, DAB will reference the configured authN provider at request time to determine which authentication scheme to use to authenticate.